onnxruntime/core/providers/openvino/ov_versions/capability.cc

```diff
@@ -0,0 +1,220 @@
+// Copyright(C) 2019 Intel Corporation
+// Licensed under the MIT License
+
+#include "core/providers/shared_library/provider_api.h"
+#include "../backend_utils.h"
+#include "../backend_manager.h"
+#include "capabilities.h"
+#include "utils.h"
+
+#if defined(_MSC_VER)
+#pragma warning(disable : 4244 4245 5208)
+#elif __GNUC__
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-parameter"
+#endif
+#include <ngraph/ngraph.hpp>
+#include <ngraph/frontend/onnx_import/onnx.hpp>
+#if defined(_MSC_VER)
+#pragma warning(default : 4244 4245)
+#elif __GNUC__
+#pragma GCC diagnostic pop
+#endif
+
+namespace onnxruntime {
+namespace openvino_ep {
+
+//Constructor 
+GetCapability::GetCapability(const GraphViewer& graph_viewer_param, std::string device_type_param,
+                             const std::string version_param):
+                graph_viewer_(graph_viewer_param), device_type_(device_type_param){
+  if (version_param == "V_2021_2") {
+    data_ops_ = new DataOps(graph_viewer_, V_2021_2, device_type_);
+  } else if (version_param == "V_2021_1") {
+    data_ops_ = new DataOps(graph_viewer_, V_2021_1, device_type_);
+  } else if (version_param == "V_2020_4") {
+    data_ops_ = new DataOps(graph_viewer_, V_2020_4, device_type_);
+  } else {
+    data_ops_ = new DataOps(graph_viewer_, V_2021_2, device_type_);
+  }
+}
+
+std::vector<std::unique_ptr<ComputeCapability>> GetCapability::Execute() {
+
+  std::vector<std::unique_ptr<ComputeCapability>> result;
+
+  //Check if it is a subgraph
+  if (graph_viewer_.IsSubgraph()) {
```

Comment:
Why isn't the capability computed if it is a subgraph?

Reply:
I have ported it directly from capability_2021_2.cc , I am not surya of the logic here